project/src/com/google/daggerquery/executor/models/Query.java

```diff
@@ -260,4 +265,56 @@ private boolean findSomePath(String source, String target, Path path,
 
     return false;
   }
+
+  /**
+   * Checks if the passed {@code node} is on the {@link BindingGraph} or if the user misspelled the node's name.
```

Comment:
Nit: 

... node} is **in** the {@link ...

Reply:
Done ✅